typescript/vrotsc/src/compiler/transformer/fileTransformers/workflow/decorators/endItemDecoratorStrategy.ts

```diff
@@ -0,0 +1,119 @@
+/*-
+ * #%L
+ * vrotsc
+ * %%
+ * Copyright (C) 2023 - 2024 VMware
+ * %%
+ * Build Tools for VMware Aria
+ * Copyright 2023 VMware, Inc.
+ *
+ * This product is licensed to you under the BSD-2 license (the "License"). You may not use this product except in compliance with the BSD-2 License.
+ *
+ * This product may include a number of subcomponents with separate copyright notices and license terms. Your use of these subcomponents is subject to the terms and conditions of the subcomponent's license, as noted in the LICENSE file.
+ * #L%
+ */
+import { Decorator, MethodDeclaration, SourceFile } from "typescript";
+import { StringBuilderClass } from "../../../../../utilities/stringBuilder";
+import { WorkflowItemDescriptor, WorkflowItemType } from "../../../../decorators";
+import { getDecoratorProps } from "../../../helpers/node";
+import CanvasItemDecoratorStrategy from "./canvasItemDecoratorStrategy";
+import { buildItemParameterBindings, InputOutputBindings } from "./helpers/presentation";
+
+// UI positioning constants in the output XML file.
+const xBasePosition = 160;
+const yBasePosition = 100;
+const offSet = 20;
+
+export default class EndItemDecoratorStrategy implements CanvasItemDecoratorStrategy {
```

Comment:
Small docs what it builds as with others?

Reply:
Documentation added.